includes/Modules/Sign_In_With_Google.php

```diff
@@ -344,32 +353,38 @@ public function get_debug_fields() {
 
 		$debug_fields = array(
 			'sign_in_with_google_client_id'                => array(
-				'label' => __( 'Sign in with Google: Client ID', 'google-site-kit' ),
+				/* translators: %s: Sign in with Google service name */
+				'label' => sprintf( __( '%s: Client ID', 'google-site-kit' ), _x( 'Sign in with Google', 'Service name', 'google-site-kit' ) ),
 				'value' => $settings['clientID'],
 				'debug' => Debug_Data::redact_debug_value( $settings['clientID'] ),
 			),
 			'sign_in_with_google_shape'                    => array(
-				'label' => __( 'Sign in with Google: Shape', 'google-site-kit' ),
+				/* translators: %s: Sign in with Google service name */
+				'label' => sprintf( __( '%s: Shape', 'google-site-kit' ), _x( 'Sign in with Google', 'Service name', 'google-site-kit' ) ),
 				'value' => $this->get_settings()->get_label( 'shape', $settings['shape'] ),
 				'debug' => $settings['shape'],
 			),
 			'sign_in_with_google_text'                     => array(
-				'label' => __( 'Sign in with Google: Text', 'google-site-kit' ),
+				/* translators: %s: Sign in with Google service name */
+				'label' => sprintf( __( '%s: Text', 'google-site-kit' ), _x( 'Sign in with Google', 'Service name', 'google-site-kit' ) ),
 				'value' => $this->get_settings()->get_label( 'text', $settings['text'] ),
 				'debug' => $settings['text'],
 			),
 			'sign_in_with_google_theme'                    => array(
-				'label' => __( 'Sign in with Google: Theme', 'google-site-kit' ),
+				/* translators: %s: Sign in with Google service name */
+				'label' => sprintf( __( '%s: Theme', 'google-site-kit' ), _x( 'Sign in with Google', 'Service name', 'google-site-kit' ) ),
 				'value' => $this->get_settings()->get_label( 'theme', $settings['theme'] ),
 				'debug' => $settings['theme'],
 			),
 			'sign_in_with_google_use_snippet'              => array(
-				'label' => __( 'Sign in with Google: One-tap Enabled', 'google-site-kit' ),
+				/* translators: %s: Sign in with Google service name */
+				'label' => sprintf( __( '%s: One-tap Enabled', 'google-site-kit' ), _x( 'Sign in with Google', 'Service name', 'google-site-kit' ) ),
 				'value' => $settings['oneTapEnabled'] ? __( 'Yes', 'google-site-kit' ) : __( 'No', 'google-site-kit' ),
 				'debug' => $settings['oneTapEnabled'] ? 'yes' : 'no',
 			),
 			'sign_in_with_google_authenticated_user_count' => array(
-				'label' => __( 'Sign in with Google: Number of users who have authenticated using Sign in with Google', 'google-site-kit' ),
+				/* translators: %s: Sign in with Google service name */
```

Comment:
I think this placeholder should be updated as well:
```suggestion
				/* translators: %1$s: Sign in with Google service name */
```